invoices/utils.py

```diff
@@ -1,7 +1,10 @@
 from django.utils.deconstruct import deconstructible
-
+from django.utils.text import get_valid_filename
```

Comment:
`get_valid_filename` is not used anymore (you removed the code that used it in a previous commit, can you also remove the import)?

Reply:
Whoops, yes!